### DISCUSSION TITLE
Unable to get IP address on the board

### DISCUSSION THREAD

**Discussion by NavSriv** [OE4T/meta-tegra#1515] [Q&A]
I am using custom board with Jetson Orin NX 16GB. I have created images using kirstone branch - Machine as p3768-0000-p3767-0000. Board comes up pretty well but on connecting LAN cable to the board, doesn't pull any ipaddress. Am I supposed to start dhcp services in some init script or is there any other easy way?
Thanks

**Comment by dwalkes** [comment]
What image are you using?
If you are just getting started I'd recommend `demo-image-base` in [tegra-demo-distro](https://github.com/OE4T/tegra-demo-distro).  This one should have everything you need for network connectivity.  I haven't personally tried `p3768-0000-p3767-0000` myself.

**Reply by NavSriv** [reply]
I am using customised image which is working similar to "core-image-sato". I am almost done with my layer and changes wrt the custom board so will probably try to make networking related changes.
